support/packaging/centos/mesos.spec

```diff
@@ -1,9 +1,6 @@
-%global MESOS_VERSION
-%global MESOS_RELEASE
-
 Name:          mesos
-Version:       %{MESOS_VERSION}
-Release:       %{MESOS_RELEASE}%{?dist}
+Version:       1.11.1
```

Comment:
This file too?

Reply:
Oh  no... Good that u look at it. :-)  This and the upper one does not have to be there. I will cleanup. Sorry! :-(